src/style.css

```diff
@@ -1,3 +1,87 @@
+@import 'https://fonts.googleapis.com/css?family=Roboto:100,100italic,300,300italic,regular,italic,500,500italic,700,700italic,900,900italic';
+
 body {
   margin: 0;
 }
+
+.header{
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  width: 100%;
+}
+
+.header__nav{
+  margin: 0 50px;
```

Comment:
1. it's padding
2. move to `header`

Reply:
when I made a padding for "header", the header was bigger than my screen